applications/nrf_desktop/src/modules/dvfs.c

```diff
@@ -49,13 +49,25 @@ static const uint8_t dvfs_medlow_freq_bitmask = BIT(DVFS_STATE_LLPM_CONNECTED);
  */
 static uint8_t dfvs_requests_state = BIT(DVFS_STATE_INITIALIZING);
 
+BUILD_ASSERT(8 * MIN(sizeof(dvfs_high_freq_bitmask),
```

Comment:
Commit message:
`Add DVFS timeout feature for nrf_desktop which allows ...`
`finnish` -> `finish`